src/v1.js

```diff
@@ -148,6 +148,42 @@ api.declare({
   res.reply(clients);
 });
 
+/** Paginate List clients */
```

Comment:
This should be a modification of the existing method, rather than a new method ([ideally](https://bugzilla.mozilla.org/show_bug.cgi?id=1449697), `api.declare` would throw an error if you declare the same route+method twice).